### DISCUSSION TITLE
请求添加禁用ipv6的选项

### DISCUSSION THREAD

**Discussion by yyzh** [HMBSbige/NatTypeTester#229] [Q&A] [answered]
测试时发现程序会优先尝试通过ipv6发送请求,而不是通过ipv4,导致无法检测ipv4下的NAT类型.
![image](https://github.com/HMBSbige/NatTypeTester/assets/4102990/89161237-0c24-40d6-a889-e47294792cf7)

**Comment by HMBSbige** [comment, marked as answer]
请求前手动更改本地地址为`0.0.0.0:端口`就行了呀
